transport_nantes/topicblog/tn_links.py

```diff
@@ -38,7 +38,7 @@
     [[don:adhésion]]((text))   equivalent to {% bouton_join "text" %}
     [[don:fixed|1]]((text))    equivalent to {% fixed_amount_donation_button 1 "text" %}
 
-    [[news:name]]((text))      equivalent to {% show_mailing_list %}
+    [[news:name]]((text))      equivalent to {% show_mailing_list name %}
```

Comment:
The news MD tag is pretty confusing in the name, I'd expect news to be somewhat related to articles (The news ..) not to show a subscribe to mailing list block.

This isn't new for the name, it's just that it made me realize it wasn't intuitive for me.

Reply:
Yes, I agree.  I want to change the name, but that seems best as a separate change, properly staged (introduce new name, deploy, make sure old is out of service, then retire old).

Reply:
Agreed